cache/edge_lambdas/src/redirector.ts

```diff
@@ -0,0 +1,89 @@
+import { CloudFrontRequestEvent, CloudFrontResponse } from 'aws-lambda';
+import { URLSearchParams } from 'url';
+import { literalRedirects, queryRedirects } from './redirects';
+
+// This is a more convenient form of the query redirects that looks like:
+// {
+//   [original path to match]: {
+//     params: [URLSearchParams to match]
+//     redirectPath: [path to redirect to]
+//   }
+// }
```

Comment:
Any reason not to have this in that shape in that file?

Reply:
Good point - no, I think I thought it was clearer like this but actually it probably isn't. Will change.